### PR TITLE
Tidy up and add accounts

### PR DESCRIPTION
**Pull request by bevanloon**
The Seal is a little out of date so...

Add the accounts team and the replatforming team.

Alphabetise datagovuk and govuk-corona-services.

Update some teams so they more accurately reflect the people on them.

I expect there's more work to get this completely accurate. Work for another day though.

### CODE REVIEW

File: config/alphagov.yml [hunk 1]
```diff
@@ -176,14 +228,16 @@ govuk-frontend-a11y:
     - "It's okay to take time to learn"
     - "It's okay not know everything"
 
-govuk-datagovuk:
+govuk-notifications:
   members:
-    - kentsanggds
-    - maxgds
-    - risicle
+    - benthorner
```
Comment (benthorner) [inline config/alphagov.yml:234]:
```suggestion
    - benthorner
    - 1pretz1
```

File: config/alphagov.yml [hunk 2]
```diff
@@ -176,14 +228,16 @@ govuk-frontend-a11y:
...
+    - kevindew
+    - laurentqro
 
   channel:
-    "#govuk-datagovuk"
+    "#govuk-notifications"
```
Comment (benthorner) [inline config/alphagov.yml]:
```suggestion
    "#govuk-notifications-dev"
```

**Review by benthorner — APPROVED** [review]
Works for me 👍

File: config/alphagov.yml (hunk 1 above)
Comment (bevanloon) [reply config/alphagov.yml:234]: 👍 good spot thanks

File: config/alphagov.yml (hunk 2 above)
Comment (bevanloon) [reply config/alphagov.yml]: yep updated thanks

**Pull request merged**